### PR TITLE
Add a "create client" command

### PR DESCRIPTION
**Pull request by wackyMole**
In response to #452 and a personal desire to stop re-writing this command in every application I build, I present to you, a create-client console command (including some basic documentation!).

Example Output:
image

I started to add an output format option/argument, but I don't know whether it would have been useful. Let me know what you think (and/or if I need to make changes).

### CODE REVIEW

File: Command/CreateClientCommand.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+namespace FOS\OAuthServerBundle\Command;
+
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+
+class CreateClientCommand extends ContainerAwareCommand
```
Comment (dkarlovi) [inline Command/CreateClientCommand.php]: @wackyMole can you add a test here?

Comment (wackyMole) [reply Command/CreateClientCommand.php]: How's that?

**Comment by dkarlovi** [comment]
LGTM. Can you change the target branch to `1.5`, I'll merge.

**Comment by dkarlovi** [comment]
Thank you @wackyMole <3

**Pull request merged**